src/app/infra/client-storage/local-storage.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import IClientStorage from './iclient-storage';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LocalStorageService implements IClientStorage {
+  set(key: string, data: unknown) {
```

Comment:
because date is of the type unknown?

Reply:
Because user can give any type to be persisted, and I can't use `any` :v

Reply:
what do you think of creating a safeAny for these cases ```[keys: string]: any```

Reply:
@iurynogueira